system-x/services/redis/src/main/java/software/tnb/redis/resource/local/LocalRedis.java

```diff
@@ -0,0 +1,58 @@
+package software.tnb.redis.resource.local;
+
+import software.tnb.common.deployment.Deployable;
+import software.tnb.redis.service.Redis;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.auto.service.AutoService;
+
+@AutoService(Redis.class)
+public class LocalRedis extends Redis implements Deployable {
+
+    private static final Logger LOG = LoggerFactory.getLogger(LocalRedis.class);
+    private static final int PORT = 6379;
+    private RedisContainer redisContainer;
+
+    @Override
+    public String defaultImage() {
+        return "redis:6.0.20-alpine";
+    }
```

Comment:
same as for ldap, it can be moved to `Redis` class

Reply:
Done